src/components/organize/tasks/forms/utils.ts

```diff
@@ -0,0 +1,45 @@
+import dayjs from 'dayjs';
+import { ZetkinTaskRequestBody } from 'types/tasks';
+
+export const isPublishedFirst = (values: ZetkinTaskRequestBody): boolean => {
```

Comment:
So fresh and
![so clean clean](https://media.giphy.com/media/yOgKZSPRYZr70QyTQ9/giphy.gif)

Reply:
i dunno lol i could have spent time making it cleaner but it does the job D:

Reply:
Oh I was being serious though. I like the well-named functions that explain exactly what they're doing.